maintenance_evolution.Rmd

```diff
@@ -201,6 +201,19 @@ You don't have to change the tests of deprecated functions until they are made d
 
 Once a function is made defunct, its tests are simply removed.
 
+## Renaming packages {#renaming-packages}
+
+Renaming a package that is in early development is fine.
+It can be the opportunity, before review, to comply with our [naming advice](#naming-your-package).
+
+Renaming a package that is already widely adopted and/or released on CRAN is problematic.
+CRAN has a [policy](https://cran.r-project.org/web/packages/policies.html) stating that Package names on CRAN are persistent and in general it is not permitted to change a package’s name.
+The package under its old name might be a dependency of packages, scripts, and feature in documentation and blog posts, etc.
```

Comment:
```suggestion
The package under its old name might be a dependency of packages, scripts, and feature in scientific publications, and blog posts, etc.
```

Reply:
We can add "scientific publications" without removing documentation.  But definitely add "scientific publications"